pkg/block/s3/walker.go

```diff
@@ -68,15 +68,15 @@ func (s *Walker) Walk(ctx context.Context, storageURI *url.URL, op block.WalkOpt
 				Address:     addr,
 				ETag:        strings.Trim(aws.ToString(record.ETag), "\""),
 				Mtime:       aws.ToTime(record.LastModified),
-				Size:        record.Size,
+				Size:        *record.Size,
```

Comment:
Can you use the aws.* helper function to extract the value as it will check for nil

Reply:
yes